openapi/pagopa-arc-be.openapi.yaml

```diff
@@ -37,7 +37,42 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorDTO'
-
+  /token/oneidentity:
+    get:
+      tags:
+        - arc auth
+      summary: "Provide authorization endpoint for exchanging code for token"
+      operationId: getAuthenticationToken
+      parameters:
+        - name: code
+          in: query
+          description: "The value used for exchanging code for token"
+          required: true
+          schema:
+            type: string
+        - name: state
+          in: query
+          description: "The value of the state this value is handled by the security component"
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: "Token Response"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TokenResponse'
+        '401':
+          description: "Wrong or missing function key"
+        '429':
+          description: "Too many Requests"
+        '500':
+          description: "Service unavailable"
```

Comment:
500 represents an `Internal Server Error`. Check 502 or 503 if you want to model that error